build-system/compile/single-pass.js

```diff
@@ -688,11 +711,30 @@ function eliminateIntermediateBundles() {
       targets.push(createFullPath(extension.version));
     }
     targets.forEach(path => {
-      const {code} = babel.transformFileSync(path, {
+      const map = loadSourceMap(path);
+      function returnMapFirst(map) {
```

Comment:
could you explain why we need this?

Reply:
Because I didn't save the map to a filesystem to load. The second param to `resorcery` is a loader callback that either has to return a sourcemap that will be remapped, or return nothing to treat the file as an original sourcemap.